source/reference/minio-server/minio-server.rst

```diff
@@ -312,6 +312,15 @@ Root Credentials
       MinIO strongly recommends specifying a unique, long, and random
       :envvar:`MINIO_ROOT_PASSWORD` value for all environments.
 
+.. envvar:: MINIO_API_ROOT_ACCESS
+
+   .. include:: /includes/common-mc-admin-config.rst
+      :start-after: start-minio-root-api-access
+      :end-before: end-minio-root-api-access
+
+   This variable corresponds to the :mc-conf:`api root_access <api.root_access>` configuration setting.
+   You can use this variable to temporarily override the configuration setting and re-enable root access to the deployment.
```

Comment:
Might be worth mentioning that envvar overrides config setting.

 Both on this page and the `mc admin config` page, I would think.

Reply:
Done